submissions/7SergSmith7/Memory-pair-game/src/js/app.js

```diff
@@ -0,0 +1,99 @@
+import "../css/vendor/normalize.css";
+import "../css/styles.css";
+
+const images = [
+  "acrobatics",
+  "boneCutter",
+  "chaosRing",
+  "doubleStrike",
+  "innerDemon",
+  "metamorphosis",
+  "splittingSoul",
+  "manaBurn",
+];
+
+const cards = [...images, ...images];
+const gameField = document.querySelector(".game-field");
+let firstCard = "";
+let secondCard = "";
+let pairsCounter = 0;
+
+function cardTemplate(image) {
+  return `<div class="card" data-name="${image}">
+ <div class="spin-card">
+   <div class="card-front">
+      <img src="./img/cardFront.png" class="front-img"/>
+    </div>
+    <div class="card-back">
+      <img class="back-img" src="./img/${image}.png" />
+   </div>
+ </div>
+</div>`;
+}
+function shuffleCards(ArrayCards) {
+  ArrayCards.sort(() => {
```

Comment:
you can simplify this function
```suggestion
  ArrayCards.sort(() => 0.5 - Math.random())
```